modules/github-repository-webhook/variables.tf

```
@@ -4,6 +4,12 @@ variable "create_github_repository_webhook" {
   default     = true
 }
 
+variable "github_base_url" {
+  description = "Github base URL to use when creating webhook (when using GitHub Enterprise)"
+  type        = string
+  default     = ""
```

```suggestion
  default     = null
```

Accoding to [the docs](https://registry.terraform.io/providers/integrations/github/latest/docs#base_url) for `base_url` it is optional, so it should be `null`.

---

Ah, sorry about that and thanks for suggestion!

---

Also, please fix the docs (to make CI green).